packages/dds/merge-tree/src/test/dirname.cts

```diff
@@ -10,4 +10,5 @@
 //   - Export '__dirname' from a .cjs file in the same directory.
 //
 // Note that *.cjs files are always CommonJS, but can be imported from ESM.
+// eslint-disable-next-line unicorn/prefer-module
```

Comment:
@tylerbutler What do you think about disabling this for test code in our base config? Unless we have imminent plans to start running our mocha tests in a browser environment, it seems reasonable to me to disable this.

Reply:
Can't think of a reason why this would be bad for test code only.

Reply:
https://github.com/microsoft/FluidFramework/pull/21814